lib/http-bridge/test/integration/Main.hs

```diff
@@ -158,6 +168,8 @@ main = hspec $ do
         -- some places where it's less annoying.
         (UseHandle h)
 
+    mkBaseUrl port = "http://localhost:" <> T.pack (show port) <> "/"
```

Comment:
You probably need `toText` here instead of `show` since we have `port :: Port` I believe.